### PR TITLE
Update boot.py

### PR DESCRIPTION
**Pull request by Saijin-Naib**
Added --pc-geometric and --use-3dmesh to more options to hopefully help boost the baseline utility of these defaults based upon some trends I've been seeing on the forum and via email.

### CODE REVIEW

**Comment by pierotofy** [comment]
Nice, thanks! :pray: 

Can we just leave orthophoto-resolution to 2.0 by default? I know some people want high resolution, but not everyone has powerful machines.

For the buildings preset I would leave `use-3dmesh` out; the 2.5D mesh tends to generate better rooftop edges (has this changed perhaps? Do you have comparisons?)

**Comment by Saijin-Naib** [comment]
Mmm, good points. Let me try and A/B test some datasets and see if my gut feeling on this pans out consistently.

**Comment by Saijin-Naib** [comment]
Forest:
image
```
Created on: 9/29/2021, 12:40:20 PM
Processing Node: node-odm-1 (manual)
Options: min-num-features: 18000, feature-quality: ultra
Average GSD: 1.07 cm
Area: 1,279.39 m²
Reconstructed Points: 650,562
```

Forest Update:  
image
```
Created on: 9/29/2021, 12:41:41 PM
Processing Node: node-odm-1 (manual)
Options: feature-quality: ultra, min-num-features: 18000, pc-geometric: true, use-3dmesh: true
Average GSD: 1.07 cm
Area: 1,279.2 m²
Reconstructed Points: 669,357
```

Definitely see more definition and leaves in the deciduous tree and most especially in the palms.

----

Buildings:  
image

image

```
Created on: 9/29/2021, 3:37:04 PM
Processing Node: Lightning - UAV4Geo (manual)
Options: mesh-size: 300000, pc-quality: high
Average GSD: 1.96 cm
Area: 4,015.58 m²
Reconstructed Points: 7,718,968
```

Buildings Update:  
image

image

```
Created on: 9/29/2021, 3:38:08 PM
Processing Node: Lightning - UAV4Geo (manual)
Options: mesh-size: 300000, pc-geometric: true, pc-quality: high, use-3dmesh: true
Average GSD: 1.96 cm
Area: 4,015.39 m²
Reconstructed Points: 7,805,299
```

Buildings is a bit more inconclusive... Overall, I think it looks more realistic, especially in the Orthophoto, but it does seem to exhibit a bit more "bleeding" around the top edge of the main structure. Perhaps fixed by feature-quality and tie-points?

**Comment by pierotofy** [comment]
> but it does seem to exhibit a bit more "bleeding" around the top edge of the main structure.

Yep. This will also get worse if you set pc-quality to anything lower than high.

Interesting result about Forests! I'm happy to add pc-geometric to Forests if it seems to improve results.

File: app/boot.py [hunk 1]
```diff
@@ -117,8 +117,6 @@ def add_default_presets():
                                                               {'name': 'mesh-size', 'value': '300000'}]})
         Preset.objects.update_or_create(name='Buildings', system=True,
                                         defaults={'options': [{'name': 'mesh-size', 'value': '300000'},
-                                                              {'name': 'use-3dmesh', 'value': True},
-                                                              {'name': 'pc-geometric', 'value': True},
```
Comment (pierotofy) [inline app/boot.py]: You can leave pc-geometric for buildings (it's probably a good one to have).

Comment (Saijin-Naib) [reply app/boot.py]: Good! I was testing that locally and it seems to help reconstruction fidelity a bit. Things are a bit less wibbly-wobbly with it on.

**Comment by pierotofy** [comment]
Looks good! Thanks @Saijin-Naib

**Comment by Saijin-Naib** [comment]
Sorry I missed re-adjustment of GSD. I thought I fixed it 😬

**Pull request merged**